src/nonLinPMMH.cpp

```diff
@@ -97,6 +99,14 @@ Rcpp::DataFrame nonLinPMMH_impl(arma::vec data, unsigned long lNumber, unsigned
                 loglike(i) = loglike(i-1);
                 logprior(i) = logprior(i-1);
             }
+            if (i % 100 == 0 && verbouse){
+		Rcpp::Rcout << "========================="<< endl;
+		Rcpp::Rcout << "Iteration number: " << i << endl;
+		Rcpp::Rcout << "The MH_ratio is: " << std :: min(1.0, MH_ratio) << endl;
```

Comment:
This would be better described as "acceptance probability" than "MH_ratio" both for accuracy and interpretability. Similarly the next item is the average acceptance probability.

Personally, I'd only bother outputting the average, but that might be a matter of taste.